digdag-plugin-utils/src/main/java/io/digdag/util/ResumableInputStream.java

```diff
@@ -0,0 +1,129 @@
+package io.digdag.util;
+//package org.embulk.spi.util;
```

Comment:
that is ok but this code is copied from org.embulk.spi.util. So technically it's better (not required) to explain about .